api/resolvers/wallet.js

```diff
@@ -64,7 +55,7 @@ export default {
         }
       })
 
-      if (wdrwl.user.id !== me.id) {
+      if (me ? wdrwl.user.id !== me.id : wdrwl.user.id !== ANON_USER_ID) {
```

Comment:
Possible privacy leak: stackers could enumerate withdrawals and find withdrawals for anons including bolt11 invoice.